File: URI_1013_the_greatest.js
const lines = ["7 14 106"];

let inputArr = lines[0].split(" ");
let val1 = Number(inputArr[0]);
let val2 = Number(inputArr[1]);
let val3 = Number(inputArr[2]);

let higher1 = higher(val1, val2); 
let higher2 = higher(higher1, val3);

console.log(higher2 + " eh o maior");

function higher(a, b) {
    return (a + b + Math.abs(a - b)) / 2;
}

File: URI_1013_the_greatest.java
import java.util.Scanner;

public class URI_1013_the_greatest {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		int val1 = scanner.nextInt();
		int val2 = scanner.nextInt();
		int val3 = scanner.nextInt();

		int higher1 = higher(val1, val2);
		int higher2 = higher(higher1, val3);

		System.out.println(higher2 + " eh o maior");
	}

	private static int higher(int a, int b) {
		return (a + b + Math.abs(a - b)) / 2;
	}
}
